CHANGELOG.md

```diff
@@ -15,7 +109,7 @@
 
     When running `wrangler dev` as an authenticated user, your requests will now run on the same servers that Cloudflare Workers run on in production. This means that what you see is what you get. `wrangler dev` should behave exactly like production, though we still recommend deploying to a staging website before going to production in order to ensure your changes are safe. This change means you get access to things like `request.cf`, the Cache API, and any Cloudflare settings you've applied in the dashboard while developing your Workers.
 
-    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [everlastingbugstopper]: https://github.com/EverlastingBugstopper
```

Comment:
idk why all of these got downcased, but they should not be. check your editor settings? it looks like a similar thing happened with `*` -> `_` and some whitespace corrections.